### PR TITLE
refactor: cleanups and updates

### PR DESCRIPTION
**Pull request by boredsquirrel**
rpm-ostree doesnt need sudo, added complete arm commands, some minor fixes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,54 +1,53 @@
 # Fedora Cosmic Atomic
-### Fedora Silverblue-based Atomic Desktop with the Pre-Alpha Cosmic Desktop Environment Included!
+### Fedora Silverblue-based Atomic Desktop with the Pre-Alpha Cosmic Desktop Environment included!
```
Comment (noelmiller) [inline README.md:2]: I prefer the capitalization here actually.

Comment (boredsquirrel) [reply README.md:2]: the header is not capitalized so I dont see the point? I am not english native so no idea how this is done normally

File: README.md [hunk 2]
```diff
@@ -1,54 +1,53 @@
...
 
-Install an rpm-ostree based desktop, like [Fedora Silverblue](https://fedoraproject.org/atomic-desktops/silverblue/) (aka atomic desktops)
+Install a Fedora Atomic Desktop, best [Fedora Silverblue](https://fedoraproject.org/atomic-desktops/silverblue/).
 
 #### Warning
-This image requires disabling SELinux. This is NOT recommended for production and is a temporary situation until this work is finished in upstream Fedora. 
+This image requires disabling SELinux. **This is NOT recommended for production** and is a temporary situation until this work is finished in upstream Fedora. 
 
-Edit `/etc/selinux/config`, set from `enforcing` to `permissive`
+    sudo setenforce 0 && getenforce
```
[the remark this answers is not in the document]
Comment (boredsquirrel) [reply README.md:19]: Setting it to permissive is the way to do this. I dont know why users should know how the config file looks like, but where to report SELinux issues would for sure be useful.

File: README.md [hunk 3]
```diff
@@ -1,54 +1,60 @@
...
+#### If you are on an ARM device
+> NOTE: Apple M-series Laptops are not supported by default Fedora ARM, use [Fedora Asahi Remix](https://asahilinux.org/fedora/)
+
+Rebase to the temporary unsigned image
+
+    rpm-ostree rebase --reboot ostree-unverified-registry:ghcr.io/ublue-os/VARIANT:40-arm64
+
+Rebase to the signed image
+
+    rpm-ostree rebase --reboot ostree-image-signed:docker://ghcr.io/ublue-os/VARIANT:40-arm64
```
Comment (castrojo) [inline README.md:47]:
```suggestion
```

Comment (castrojo) [reply README.md:47]: We don't have arm images so we don't need this.

**Review by castrojo — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (boredsquirrel) [reply README.md:47]: lol this was already there, so I will remove the whole part

**Review by castrojo — APPROVED** [review]

**Review by boredsquirrel — COMMENTED** [review]
is this how to review changes?

**Review by castrojo — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1,54 +1,49 @@
...
+> The COSMIC Desktop Environment is still PRE ALPHA. Do not daily drive this image on your main workstation unless you know what you're doing.
 
 Like Fedora? Want to try the latest from the work in progress Cosmic Desktop Environment? Want to help find bugs and/or contribute to Cosmic development, but don't want to work in a VM or install Pop!_OS? None of the above things but something else??!?
 
 Go ahead and try one of the ostree images I've created here!
 
 ### Quick Installation
 
-Install an rpm-ostree based desktop, like [Fedora Silverblue](https://fedoraproject.org/atomic-desktops/silverblue/) (aka atomic desktops)
+Install a Fedora Atomic Desktop, best [Fedora Silverblue](https://fedoraproject.org/atomic-desktops/silverblue/).
```
Comment (dylanmtaylor) [inline README.md:14]: best Fedora Silverblue? 

This wording should be fixed back to 'like'.

File: README.md [hunk 5]
```diff
@@ -1,54 +1,49 @@
...
 
 #### Warning
-This image requires disabling SELinux. This is NOT recommended for production and is a temporary situation until this work is finished in upstream Fedora. 
+This image requires disabling SELinux. **This is NOT recommended for production** and is a temporary situation until this work is finished in upstream Fedora. 
 
-Edit `/etc/selinux/config`, set from `enforcing` to `permissive`
+    sudo setenforce 0 && getenforce
 
-> Note: Rebase to an unsigned image before rebasing to the signed cosmic images below!
+You can view the selinux config in `/etc/selinux/config`
```
Comment (dylanmtaylor) [inline README.md:21]: SELinux

File: README.md (hunk 4 above)
Comment (boredsquirrel) [reply README.md:14]: this was given the fact that it was previously a Fedora silverblue based image. will revert it back if you think that makes sense.

**Pull request merged**